_includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md

```diff
@@ -282,6 +282,25 @@ $config[AvailabilityNotificationConstants::STORE_TO_YVES_HOST_MAPPING] = [
 ];
 ```
 
+2.To set YVES host environment variables, adjust a Deploy file:
```

Comment:
Use the real Deploy file name.
```suggestion
2. To set YVES host environment variables, adjust the `deply.*.yml` file:
```

Reply:
We have many deploy files. Also customer can have it's own deploy file names